### ISSUE TITLE
feature request compat with irons spells n spellbooks mana

### ISSUE THREAD

**Issue by tenzedkariyu19** [Dplayend/Just-Leveling#6]
compat with the magic level that increases mana from https://www.curseforge.com/minecraft/mc-mods/irons-spells-n-spellbooks

**Comment by Dplayend** [comment]
you want a passive to increases mana?

**Comment by kaairu** [comment]
yeah as you UG MAG you get like 10 or wtv configured of max mana added

**Comment by VirtualGaming1** [comment]
same thing for this as well please 
https://www.curseforge.com/minecraft/mc-mods/ars-nouveau
I am using 1.18.2 version
